src/cli/cli.go

```diff
@@ -24,6 +24,7 @@ func (c CLI) newProject(cCtx *cli.Context) project.Project {
 		cCtx.String("project-directory"),
 		cCtx.String("runtime-version"),
 		cCtx.String("app-version"),
+		cCtx.String("docker-image"),
```

Comment:
We don't need this, we already know which image we have built based on the other parameters

Reply:
Spoke with @zimny about this in the stand up earlier. Currently the `deploy` command does not have the same parameters as the `build` command, so they are acting independently, which makes sense if we want them as two separate commands. I've updated the description above:

> This only partially fixes the issue as currently the `deploy` command is not dependant on `build`. To completely fix this, we'll need to refactor these commands so that the DockerService and other services can be injected as dependancies when creating implementations of `cli.Command`. This way once the `build` command is finished, it will populate the `docker-image` flag given in this step and deploy it with said container allowing the commands to be run separately if needed.

I was going to put this as on big PR but we thought it best to separate them out into two PR's